### PR TITLE
Ta161544/quiz filter accessibility

### PR DESCRIPTION
**Pull request by bemailloux**
🍞 

also, adds a tooltip mentioning that quiz filter has an effect on OU filter.

toast

Quality notes
* Testing
  * Toast message appears when new search input is entered
  * Toast message appears when search input is cleared
  * Toast message does not appear if search input is applied or cleared repeatedly
  * Screen reader (NVDA x Chrome):
    * When focused, screen reader reads out search input label, placeholder, current value and tooltip contents
    * When input is entered, screen reader reads out new dashboard status and toast message
* Security, perf, docs, devops: N/A
* UX:
  * Accessibility: see above
  * Demo: done

### CODE REVIEW

File: locales/en.js [hunk 1]
```diff
@@ -213,6 +213,8 @@ export default {
 	"table:sortedDescending": "Sorted Descending",
 	"table:flaggedRowToolTip": 'Action may be required.',
 
-	"quizNameFilter:label": "Search quizzes by name",
-	"quizNameFilter:placeholder": "Search by Quiz Name..."
+	"quizNameFilter:label": "Search quizzes by name.",
+	"quizNameFilter:placeholder": "Search by Quiz Name...",
+	"quizNameFilter:filterName": "Quiz Search",
+	"quizNameFilter:helpInfo": "Searching by quiz name will affect available options in the Org Unit filter."
```
Comment (bbabu01) [inline locales/en.js:219]: nit: Should text 'quiz name' be in title case to match placeholder

**Review by bbabu01 — APPROVED** [review]
LGTM

File: locales/en.js (hunk 1 above)
Comment (bemailloux) [reply locales/en.js:219]: Maybe, I still have to check wording with Megan (this isn't something she asked for but I expect users will wonder what is going on with OU filter if we don't explicitly tell them)

**Review by Vitalii-Misechko — APPROVED** [review]
LGTM!

**Pull request merged**